ResourceStatusSystem/scripts/dirac-rss-sync.py

```diff
@@ -127,10 +135,9 @@ def initSites():
     Initializes Sites statuses taking their values from the "SiteMask" table of "JobDB" database.
   '''
 
-  jobClient = JobDB()
-  rssClient = ResourceStatusDB()
+  rssClient = ResourceStatusClient.ResourceStatusClient()
```

Comment:
Ah, I see. I did not pay attention that this client is not inheriting the base Client class. Is there a special reason for that ? With this inheritance all the service methods are immediately available as if they implemented in the client. Instead, there is an internal redirection which is doing essentially the same thing.